blocks/section-title-block/features/section-title/_children/section-title.scss

```diff
@@ -1,14 +1,16 @@
-h1.section-title {
+.section-title {
   color: $ui-primary-font-color;
   display: block;
+  font-weight: bold;
 }
 
 .section-container {
   margin-top: map-get($spacers, 'sm');
+}
 
-  > a {
-    color: $ui-primary-font-color;
-    font-size: 14px;
-  }
+.section-title--styled-link {
+  font-size: 14px;
```

Comment:
Seeing as we are touching this line in this ticket, we should update to use `rem` sizing rather than `px`

Reply:
@matthewroach updated!

Reply:
I forget about all of our new initiatives